libraries/wallet/wallet.cpp

```diff
@@ -302,6 +302,8 @@ class wallet_api_impl
 public:
    api_documentation method_documentation;
 private:
+   const int64_t transaction_expiration_time_sec = 30;
```

Comment:
please change this to `uint8_t`

Reply:
No, `uint8_t` would be too small.
By the way I think the variable name should be "default_something".
However, if we refactored the other 2 functions, we'll no longer need a constant.

Reply:
ok, if used lets make it `uint16_t` then. i think `int64_t` is too much.

Reply:
I think there is no reason to save 6bytes of memory.
This isn't written to blockchain or held 1billion times in memory.

Reply:
I think it is not necessary to have `int64_t` to hold `30` or a small value on it but it i suppose it is just a matter of personal taste. I agree it will not save us pretty much anything in this case and if the block changes as suggested by @abitmore  this will not be even used so no point on discussing it too much.